### PR TITLE
Update DeformableConvolution

### PR DESCRIPTION
**Pull request by DingZhangIntel**

### CODE REVIEW

File: ngraph/core/reference/include/ngraph/runtime/reference/deformable_convolution.hpp [hunk 1]
```diff
@@ -99,14 +99,16 @@ namespace ngraph
...
                                           const T* batch,
                                           const Shape& batch_shape,
                                           const T* offsets,
                                           const Shape& offset_shape,
                                           const T* filter,
                                           const Shape& filter_shape,
-                                          T* out)
+                                          T* out,
+                                          size_t group_idx,
+                                          int64_t groups,
```
Comment (YangleiZouIntel) [inline ngraph/core/reference/include/ngraph/runtime/reference/deformable_convolution.hpp:110]: Does the ref implementation just have not implemented group parameter yet? or it has incorrect implementation of group size?

Comment (DingZhangIntel) [reply ngraph/core/reference/include/ngraph/runtime/reference/deformable_convolution.hpp:110]: The ref implementation uses the group parameter on line 196.

Comment (DingZhangIntel) [reply ngraph/core/reference/include/ngraph/runtime/reference/deformable_convolution.hpp:110]: The error is mainly about the deformable_groups parameter.

**Comment by YangleiZouIntel** [comment]
submitted to upstream.

**Pull request closed without merging**